keyboards/reviung/reviung46/keymaps/via/keymap.c

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 gtips (@gtips)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+	_BASE,
+	_LOWER,
+	_RAISE,
+	_ADJUST
+};
+
+#define LOWER  MO(_LOWER)
+#define RAISE  MO(_RAISE)
+#define ADJUST MO(_ADJUST)
+
+#define SF_SS RSFT_T(KC_SLSH)
+#define SP_LO LT(LOWER, KC_SPC)
+#define SP_RA LT(RAISE, KC_SPC)
```

Comment:
```suggestion
#define SP_LO LT(_LOWER, KC_SPC)
#define SP_RA LT(_RAISE, KC_SPC)
```

Reply:
Thanks.
I fixed it.
It worked fine on the test board.